Ports/qt6-qtbase/patches/0001-Add-a-SerenityOS-platform-definition.patch

```diff
@@ -148,7 +148,7 @@ index 0000000000000000000000000000000000000000..1f0dd9819abb1f8dac52250d24aa8d07
 +#endif
 +
 +#include <langinfo.h>
-+#include <sys/ioctl_numbers.h>
++#include <Kernel/API/Ioctl.h>
```

Comment:
My preference would be to include <sys/ioctl.h> and reserve Kernel/API headers for lower-level libraries like LibC to include.

Reply:
Changed it.